stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -372,6 +375,32 @@ private void echo(String msg) {
         log.info(msg);
     }
 
+    private void copyUploadDirectoryToAgents() throws IOException {
+        log.info("WORKING DIRECTORY:" + WORKING_DIRECTORY.getAbsolutePath());
```

Comment:
PS: The name of the field should be in caps, because it is a constant. But why are you using caps in the logging?